src/quequations.jl

```diff
@@ -1,61 +1,167 @@
 abstract QuEquation
 
+@doc """
+Schrodinger Equation type
+
+Fields :
+
+`hamiltonian` : Hamiltonian of the system is the only field for the type.
+""" ->
 immutable QuSchrodingerEq{H<:QuBase.AbstractQuMatrix} <: QuEquation
     hamiltonian::H
     QuSchrodingerEq(hamiltonian) = new(hamiltonian)
 end
 
+@doc """
+Schrodinger Equation method
+
+Input Parameters :
+
+`hamiltonian` : Hamiltonian of the system to construct `QuSchrodingerEq` type.
+""" ->
 QuSchrodingerEq{H<:QuBase.AbstractQuMatrix}(hamiltonian::H) = QuSchrodingerEq{H}(hamiltonian)
 
+@doc """
+Liouville von Neumann Equation type
+
+Fields :
+
+`liouvillian` : Liouvillian of the system is the only field for the type.
+""" ->
 immutable QuLiouvillevonNeumannEq{H<:QuBase.AbstractQuMatrix} <: QuEquation
     liouvillian::H
     QuLiouvillevonNeumannEq(liouvillian) = new(liouvillian)
 end
 
+@doc """
+Liouville von Neumann Equation method
+
+Input Parameters :
+
+`liouvillian` : Liouvillian of the system to construct `QuLiouvillevonNeumannEq` type.
+""" ->
 QuLiouvillevonNeumannEq{H<:QuBase.AbstractQuMatrix}(liouvillian::H) = QuLiouvillevonNeumannEq{H}(liouvillian)
 
-function liouvillian_op(h::QuBase.AbstractQuMatrix)
-    nb = size(coeffs(h), 1)
+@doc """
+Lindblad Master Equation type
+
+Fields :
+
+`lindblad`      : Lindblad operator of the system
+`hamiltonain`   : Hamiltonian of the system
+`collapse_ops`  : Collapse operators
+""" ->
+immutable QuLindbladMasterEq{L<:QuBase.AbstractQuMatrix, H<:QuBase.AbstractQuMatrix, V<:QuBase.AbstractQuMatrix} <: QuEquation
+    lindblad::L
+    hamiltonian::H
+    collapse_ops::Vector{V}
+    QuLindbladMasterEq(lindblad, hamiltonian, collapse_ops) = new(lindblad, hamiltonian, collapse_ops)
+end
+
+@doc """
+Lindblad Master Equation method
+
+Input Parameters :
+
+`hamiltonian` : Hamiltonain of the system
+`collapse_ops` : Collapse operators
+""" ->
+QuLindbladMasterEq{H<:QuBase.AbstractQuMatrix, V<:QuBase.AbstractQuMatrix}(hamiltonian::H, collapse_ops::Vector{V}) = QuLindbladMasterEq{typeof(lindblad_op(hamiltonian, collapse_ops)),H,V}(lindblad_op(hamiltonian, collapse_ops), hamiltonian, collapse_ops)
```

Comment:
This is a very expensive way since you have to compute the lindblad_op twice (and this operation might take a substantial amount of time)! Just do it like this
```
function QuLindbladMasterEq{H<:QuBase.AbstractQuMatrix, V<:QuBase.AbstractQuMatrix}(hamiltonian::H, collapse_ops::Vector{V}) 
    lop = lindblad_op(hamiltonian, collapse_ops)
    return QuLindbladMasterEq{typeof(lop),H,V}(lop, hamiltonian, collapse_ops)
end
```